pilosa.go

```diff
@@ -70,11 +70,7 @@ var (
 	ErrTooManyWrites    = errors.New("too many write commands")
 
 	ErrConfigClusterTypeInvalid = errors.New("invalid cluster type")
-	ErrConfigClusterTypeMissing = errors.New("missing cluster type")
 	ErrConfigHostsMissing       = errors.New("missing bind address in cluster hosts")
-	ErrConfigBroadcastPort      = errors.New("internal-port not found in internal-hosts")
-	ErrConfigHostsMismatch      = errors.New("hosts and internal-hosts length mismatch")
-	ErrConfigReplicaNInvalid    = errors.New("replica number must be <= hosts")
```

Comment:
I think we should keep the ErrConfigReplicaNInvalid validation check unless there is a valid reason to have a replica number > the number of hosts.